packages/web/docs/src/components/feature-tabs.tsx

```diff
@@ -128,7 +128,12 @@ export function FeatureTabs({ className }: { className?: string }) {
         </Tabs.List>
         <div className="grid grid-cols-1 lg:grid-cols-2">
           <>
-            <Tabs.Content value="Schema Registry" tabIndex={-1}>
+            <Tabs.Content
+              value="Schema Registry"
+              forceMount
+              className="data-[state=inactive]:hidden"
```

Comment:
```suggestion
              {/* Make it accessible to crawlers, otherwise there's no DOM element */}
              forceMount
              className="data-[state=inactive]:hidden"
```